### PR TITLE
SEO-173277 ASP.NET Core Description too long too short

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK ,
I have made changes in the description section, please review.
Regards,
Scholar.

### CODE REVIEW

File: aspnet-core/Menu/Context-Menu.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Context Menu | Menu | ASP.NET Core | Syncfusion
+title: Context Menu in ASP.NET Core Menu | Syncfusion
```
Comment (MallikaRK) [inline aspnet-core/Menu/Context-Menu.md:3]: Context Menu in ASP.NET Core | Syncfusion

Change the title as per above.

File: aspnet-core/PivotGrid/Getting-Started.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: Getting-Started | PivotGrid | ASP.NET Core | Syncfusion
-description: Learn here about getting started with PivotGrid in Syncfusion Essential ASP.NET Core Control, its elements, and more.
+title: Getting-Started in ASP.NET Core PivotGrid | Syncfusion
```
Comment (MallikaRK) [inline aspnet-core/PivotGrid/Getting-Started.md:3]: Getting-Started | ASP.NET Core PivotGrid | Syncfusion

change the title.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: aspnet-core/PivotGrid/Getting-Started.md (hunk 2 above)
Comment (Scholar-jaruha) [reply aspnet-core/PivotGrid/Getting-Started.md:3]: changed

File: aspnet-core/NumericTextBox/Overview.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: NumericTextBox | NumericTextBox | ASP.NET Core | Syncfusion
+title: Overview in ASP.NET Core NumericTextBox | Syncfusion
```
[the remark this answers is not in the document]
Comment (Scholar-jaruha) [reply aspnet-core/NumericTextBox/Overview.md:3]: changed

File: aspnet-core/Menu/Context-Menu.md (hunk 1 above)
Comment (Scholar-jaruha) [reply aspnet-core/Menu/Context-Menu.md:3]: changed

**Review by MallikaRK — APPROVED** [review]
Approved

**Pull request merged**